widget/label.go

```diff
@@ -59,29 +57,59 @@ func (l *Label) Bind(data binding.String) {
 	data.AddListener(l.textListener)
 }
 
-// Refresh checks if the text content should be updated then refreshes the graphical context
+// CreateRenderer is a private method to Fyne which links this widget to its renderer
+func (l *Label) CreateRenderer() fyne.WidgetRenderer {
+	l.provider = NewRichTextWithText(l.Text)
+	l.ExtendBaseWidget(l.super())
+	l.syncSegments()
+
+	return l.provider.CreateRenderer()
+}
+
+// ExtendBaseWidget is used by an extending widget to make use of BaseWidget functionality.
+func (l *Label) ExtendBaseWidget(w fyne.Widget) {
+	if w == nil {
+		w = l
+	}
+	l.BaseWidget.ExtendBaseWidget(w)
+	if l.provider != nil {
+		l.provider.ExtendBaseWidget(w)
+	}
+}
+
+// MinSize returns the size that this label should not shrink below.
+//
+// Implements: fyne.Widget
+func (l *Label) MinSize() fyne.Size {
+	if l.provider == nil {
+		l.CreateRenderer()
+	}
```

Comment:
Although this apparently fixes the problem before, this now creates a "phantom" renderer, this should be re-written to:
```go
	if l.provider == nil {
		l.ExtendBaseWidget(l)
		cache.Renderer(l.super())
	}
```